### PR TITLE
Convert to ts: Nav, Private and PrivateROute.js done

### PR DESCRIPTION
**Pull request by DavielSA**
Done.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ debug.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+bash.exe.stackdump
```
Comment (dailymp) [inline .gitignore]: Maybe you should add ```*.exe.*```

File: src/Private.tsx [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+import * as React from "react";
+import Auth from "./Auth/Auth";
 
-class Private extends Component {
+interface Props {
+  auth: Auth;
+  history: History;
+}
+interface State {
+  // tokenRenewalComplete: boolean;
```
Comment (dailymp) [inline src/Private.tsx]: Remove this and all commented code please

File: src/Private.tsx [hunk 3]
```diff
@@ -1,15 +1,24 @@
-import React, { Component } from "react";
+import * as React from "react";
+import Auth from "./Auth/Auth";
 
-class Private extends Component {
+interface Props {
+  auth: Auth;
+  history: History;
```
Comment (dailymp) [inline src/Private.tsx]: Delete this unused prop.

File: src/PrivateRoute.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+
+import { Route } from "react-router-dom";
+import AuthContext from "./AuthContext";
+
+interface Props {
+  path:string;
+  component:  typeof React.Component;
+  scopes:string[];
```
Comment (dailymp) [inline src/PrivateRoute.tsx]: You should format documents please use shift + alt +f

File: .gitignore (hunk 1 above)
Comment (DavielSA) [reply .gitignore]: ok. Done.

File: src/Private.tsx (hunk 2 above)
Comment (DavielSA) [reply src/Private.tsx]: ok. Done.

File: src/Private.tsx (hunk 3 above)
Comment (DavielSA) [reply src/Private.tsx]: ok. Done.

File: src/PrivateRoute.tsx (hunk 4 above)
Comment (DavielSA) [reply src/PrivateRoute.tsx]: ok. Done.

**Pull request merged**